src/connector/connection_info.rs

```diff
@@ -260,6 +226,7 @@ impl fmt::Display for SqlFamily {
 
 #[cfg(test)]
 mod tests {
+    #[cfg(feature = "sqlite")]
```

Comment:
why only on sqlite?

Reply:
When you compile without sqlite feature, this gives a warning.